packages/connect/src/device/DeviceCommands.ts

```diff
@@ -693,6 +693,16 @@ export class DeviceCommands {
             };
         }
 
+        if (coinInfo.shortcut === 'SOL') {
```

Comment:
btw is there some kind of solana testnet?

Reply:
Yes there is, I choose to implement mainnet first to avoid some testnet specific issues and work with the production data right from the beginning. My plan is to add testnet in the end.